packages/server/lib/cloud/api/put_protocol_artifact.ts

```diff
@@ -25,23 +23,27 @@ export const putProtocolArtifact = asyncRetry(
       throw new Error(`Spec recording too large: artifact is ${size} bytes, limit is ${maxFileSize} bytes`)
     }
 
-    const activityMonitor = new StreamActivityMonitor(MAX_START_DWELL_TIME, MAX_ACTIVITY_DWELL_TIME)
+    const activityMonitor = new StreamActivityMonitor(MAX_ACTIVITY_DWELL_TIME)
     const fileStream = fs.createReadStream(artifactPath)
     const controller = activityMonitor.getController()
 
-    await putFetch(destinationUrl, {
-      parse: ParseKinds.TEXT,
-      headers: {
-        'content-length': String(size),
-        'content-type': 'application/x-tar',
-        'accept': 'application/json',
-      },
-      // ts thinks this is a web fetch, which only expects ReadableStreams.
-      // But, this is a node fetch, which supports ReadStreams.
-      // @ts-expect-error
-      body: activityMonitor.monitor(fileStream),
-      signal: controller.signal,
-    })
+    try {
+      await putFetch(destinationUrl, {
+        parse: ParseKinds.TEXT,
+        headers: {
+          'content-length': String(size),
+          'content-type': 'application/x-tar',
+          'accept': 'application/json',
+        },
+        // ts thinks this is a web fetch, which only expects ReadableStreams.
+        // But, this is a node fetch, which supports ReadStreams.
+        // @ts-expect-error
+        body: activityMonitor.monitor(fileStream),
+        signal: controller.signal,
+      })
+    } catch (e) {
+      throw e
```

Comment:
whats the point of catching to immediately rethrow?

Reply:
I had some temporary debugging in there - will remove

Reply:
addressed in https://github.com/cypress-io/cypress/pull/29959/commits/d302a052dacd5b05322bff32737992da28022f2f